gobblin-runtime/src/main/java/gobblin/runtime/api/JobCatalog.java

```diff
@@ -30,7 +31,7 @@
  * A catalog of all the {@link JobSpec}s a Gobblin instance is currently aware of.
  */
 @Alpha
-public interface JobCatalog extends JobCatalogListenersContainer, Instrumentable {
+public interface JobCatalog extends JobCatalogListenersContainer, Instrumentable, Service {
```

Comment:
As discussed offline, let's not force every catalog be a service. StandardGobblinInstanceDriver takes care of dynamically checking for components that are services and starting them automatically.